website/versioned_docs/version-0.5/navigation.md

```diff
@@ -12,22 +12,86 @@ If you'd like to achieve a native look and feel on both Android and iOS, or you'
 
 ## React Navigation
 
-The community solution to navigation is a standalone library that allows developers to set up the screens of an app with a few lines of code.
+The community solution to navigation is a standalone library that allows developers to set up the screens of an app with a few lines of code. Assuming you are on the latest react-native version.
 
 The first step is to install in your project:
 
 ```
-npm install --save react-navigation
+yarn add react-navigation
+# or with npm
+# npm install react-navigation
 ```
 
-The second step is to install react-native-gesture-handler
+The second step is to install additional dependencies such as `react-native-gesture-handler`, `react-native-reanimated`, and `react-native-screens`:
```

Comment:
and `react-native-safe-area-context` and `masked-view`

Reply:
okay, and alredy joined into one line here -> 571f5ed1056eccac556a608256363587d9993674